src/test/scala/scalafix/internal/sbt/SbtCompletionsSuite.scala

```diff
@@ -25,7 +25,7 @@ class SbtCompletionsSuite extends AnyFunSuite {
 
   val exampleDependency = {
     import sbt._
-    "ch.epfl.scala" %% "example-scalafix-rule" % "1.4.0"
+    "ch.epfl.scala" %% "example-scalafix-rule" % "2.0.0-RC1"
```

Comment:
necessary to avoid getting the noise of the [compatibility warning](https://github.com/scalacenter/scalafix/blob/afced0705ff246850c5e7cc32698509dd4aa30a0/scalafix-cli/src/main/scala/scalafix/internal/interfaces/ScalafixArgumentsImpl.scala#L111-L115) in stdout